src/Aquifer.Jobs/Program.cs

```diff
@@ -15,6 +15,14 @@
 
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults()
+    .ConfigureAppConfiguration((context, builder) => builder
+        .SetBasePath(context.HostingEnvironment.ContentRootPath)
+        .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
```

Comment:
Does adding the new `appsettings.json` stop the old `local.settings.json` from loading in?

Reply:
No, all of the local settings values get populated as previous